client/scss/components/_figure.scss

```diff
@@ -54,37 +54,49 @@
   }
 }
 
-$large-width: (5/7 * 100)+'%';
+$narrow-width: (5/7) * 100%; // calculates the narrow column width as a percentage of the main column.
+$center-width-large: (6/7) * 100%; // calculates the halfway point of the container, as a percentage of the main column.
+$center-width-xlarge: (6/6) * 100%;
 
 .figure--full-width {
```

Comment:
We'll need to pay heed to this [here](https://github.com/wellcometrust/wellcomecollection.org/pull/285/files#diff-28980bbc8de92d1bd1580cda008f8b1aR51) - but hopefully we can get one class to use for both.